cmd/podman/trust.go

```diff
@@ -144,31 +143,12 @@ func showTrustCmd(c *cli.Context) error {
 	}
 
 	sortedRepos := sortPolicyJSONKey(policyJSON)
```

Comment:
It’s a bit surprising that keys of `policyJSON` are used to sort `showOutputMap`. Why not do the sort directly on the keys of `showOutputMap`?